### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by Tamila-Bodnar**
Adding material changed many files

### CODE REVIEW

File: src/app/core/components/login/login.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (rmurtazin) [inline src/app/core/components/login/login.component.html:1]: Remove this file, Maria implemented it

File: src/app/core/components/login/login.component.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.container {
```
Comment (rmurtazin) [inline src/app/core/components/login/login.component.scss:1]: Remove this file also

File: src/app/core/components/login/login.component.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
```
Comment (rmurtazin) [inline src/app/core/components/login/login.component.ts:1]: Remove whole login component

File: src/app/core/services/authentication.service.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService {
+  TOKEN_KEY = 'token_key';
+  isAuthenticated = false;
```
Comment (rmurtazin) [inline src/app/core/services/authentication.service.ts:9]: why we can't use only checkAuthentication() method only?

**Comment by Tamila-Bodnar** [comment]
Create new branch routing-for-merge

**Pull request closed without merging**